packages/labs/compiler/tsconfig.json

```diff
@@ -18,10 +18,9 @@
     "noFallthroughCasesInSwitch": true,
     "noImplicitAny": true,
     "noImplicitThis": true,
-    "moduleResolution": "NodeNext",
-    "stripInternal": true,
+    "moduleResolution": "Node",
```

Comment:
Curious as to why this was needed as many of our packages use `"NodeNext"` which I think is preferable as it's needed for resolving package exports.

Reply:
I haven't figured out why this is needed. If I use `NodeNext` I get the errors.
(I get lots, but this is the main error, repeated):
```
../../../node_modules/@parse5/tools/lib/creation.d.ts:2:91 - error TS2307: Cannot find module 'parse5/dist/tree-adapters/default.js' or its corresponding type declarations.

2 import type { CommentNode, TextNode, Element, Template, DocumentFragment, Document } from 'parse5/dist/tree-adapters/default.js';
                                                                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I see, thank you for the details. Looks like the types for `@parse5/tools` is digging into the `parse5` package's innards not listed in the `package.json`. This is an upstream issue that seems fixable but not important for now.

Reply:
Filed issue here: https://github.com/parse5/parse5-tools/issues/16

Reply:
Thank you so much!